data/abilities.js

```diff
@@ -3530,8 +3529,7 @@ let BattleAbilities = {
 			if (effect && effect.id === 'toxicspikes') return;
 			if (status.id === 'slp' || status.id === 'frz') return;
 			this.add('-activate', target, 'ability: Synchronize');
-			// @ts-ignore
-			source.trySetStatus(status, target, {status: status.id, id: 'synchronize'});
+			source.trySetStatus(status, target, Object.assign({status: status.id}, this.effect));
```

Comment:
Please don't make the code slower for TypeScript.

This is why `ts-ignore` exists. :|

(Although I would just cast here)